Anytype/Sources/PresentationLayer/Auth/JoinFlow/VaultView/VaultViewModel.swift

```diff
@@ -45,7 +45,7 @@ final class VaultViewModel: ObservableObject {
             do {
                 state.mnemonic = try await authService.createWallet()
                 let account = try await authService.createAccount(
-                    name: state.soul,
+                    name: Loc.myFirstSpace,
```

Comment:
we use `state.soul` on the last screen of onboarding, should it be prefilled with `Loc.myFirstSpace`? I think so